libraries/AP_GPS/AP_GPS_ERB.h

```diff
@@ -24,6 +24,10 @@
 #include "AP_GPS.h"
 #include "GPS_Backend.h"
 
+#ifndef AP_GPS_ERB_ENABLED
+  #define AP_GPS_ERB_ENABLED 1
+#endif
+
```

Comment:
```suggestion
#if AP_GPS_ERB_ENABLED
```

... if this is too hard it can be done later, but really helps ensure that everything related to the driver has been removed

similarly elsewhere

Reply:
okay did it in every .h file